application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -56,20 +58,18 @@ public HelpThreadCreatedListener(HelpSystemHelper helper) {
     }
 
     @Override
-    public void onChannelCreate(ChannelCreateEvent createEvent) {
-        if (!createEvent.getChannelType().isThread()) {
-            return;
-        }
-        ThreadChannel threadChannel = createEvent.getChannel().asThreadChannel();
-
-        if (wasThreadAlreadyHandled(threadChannel.getIdLong())) {
-            return;
-        }
-
-        if (!helper.isHelpForumName(threadChannel.getParentChannel().getName())) {
-            return;
+    public void onMessageReceived(@NotNull MessageReceivedEvent event) {
+        if (event.isFromThread()) {
+            Channel parentChannel = event.getChannel().asThreadChannel().getParentChannel();
+            if (helper.isHelpForumName(parentChannel.getName())) {
+                ThreadChannel threadChannel = event.getChannel().asThreadChannel();
+                int messageCount = threadChannel.getMessageCount();
+                if (messageCount > 1 || wasThreadAlreadyHandled(threadChannel.getIdLong())) {
```

Comment:
messageCount is reliable, remove wasThreadAlreadyHandled

Reply:
It's not, I already had a chit-chat with waz regarding another solution where we have to pull an updated history of thread on creation to rely on message count. However since wasAlreadHandled also works, I decided to not change it. 

You can try it out locally by removing it.